ignition-math/PKGBUILD

```diff
@@ -1,7 +1,7 @@
 # Maintainer: acxz <akashpatel2008 at yahoo dot com>
 # Contributor: Mykola Dolhyi <0xb000@gmail.com>
 pkgname=ignition-math
-pkgver=6.4.0
+pkgver=7.0.0
```

Comment:
Mmmh only problem is, when using 6.4.0 instead of master, the build breaks for the same reason it broke on Bitbucket. The fix for this bug was only pushed 6 days ago, i.e. on the current master branch

Reply:
hmm in this case we will have to patch the package with that commit. If you are interested in doing the fix take a look at https://wiki.archlinux.org/index.php/Patching_packages or I could take care of it later today prob,

Or someone could create a ignition-math-git package and use that.

Reply:
Yea, I'll create the patch!